app/controllers/sessions_controller.rb

```diff
@@ -16,6 +16,7 @@ def create
 
     if resource.valid_password?(params[:password])
       sign_in :user, resource
+      resource.update!(lastActive: Time.current)
```

Comment:
I would probably use `update_column` here instead of `update!`, unless there is a reason we would want to run callbacks and validations to update this field?

Reply:
It looks like this is only used for PropertyManagers, I guess there is no harm in updating it for all types of users.

I believe this is more of a last_sign_in rather than a last_active timestamp, which might be a little bit different than how it was originally intended, but I think that is ok. If it needs to change later we can do that.